src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxButton.java

```diff
@@ -169,12 +198,15 @@ public void paintComponent(final Graphics g) {
         }
     }
 
-    protected void doRendererPaint(final Graphics g, final ButtonModel buttonModel, final boolean editable, final Insets insets, int left, int top, int width, int height) {
+    final Component getRendererComponent() {
```

Comment:
```suggestion
    private Component getRendererComponent() {
```

I wanted to use this method in `AquaComboBoxUI` but eventually I added `updateAccessibleName` instead, thus this method can be made `private`.

Reply:
Updated.